extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -127,6 +127,21 @@ export function isPositionFixed(el, win) {
   return hasFixedAncestor;
 }
 
+/**
+ * @param {!Element} el
+ * @param {!Window} win
+ * @return {string|null} a string of the name of valid containers of the ad.
+ * This is called before creating iframe for this ad.
+ */
+export function getAdContainer(el) {
+  do {
```

Comment:
Does `sticky-ad` and `flying-carpet` accepts child other than `amp-ad`? If not, we don't need the loop for now. We can add when it's really needed.